### PR TITLE
Teva Digihaler Redirection

### PR DESCRIPTION
**Pull request by hastagurung**

### CODE REVIEW

**Review by bidur-phil — APPROVED** [review]

File: netlify.toml [hunk 1]
```diff
@@ -91,6 +91,11 @@ to = "https://phil.us/:splat"
 status = 301
 force = true
 
+[[redirects]]
+from = "/digihaler/*"
+to = "https://enroll.phil.us/digihaler"
+status = 200
```
Comment (JwalantaPhil) [inline netlify.toml:97]: this should be 301 (permanent redirect)

File: netlify.toml [hunk 2]
```diff
@@ -91,6 +91,11 @@ to = "https://phil.us/:splat"
 status = 301
 force = true
 
+[[redirects]]
+from = "/digihaler/*"
```
Comment (JwalantaPhil) [inline netlify.toml:95]:
```suggestion
from = "/digihaler"
```

File: netlify.toml [hunk 3]
```diff
@@ -91,6 +91,11 @@ to = "https://phil.us/:splat"
...
+to = "https://enroll.phil.us/digihaler"
```
[the remark this answers is not in the document]
Comment (hastagurung) [reply netlify.toml:96]: After submitting the form, the page will be redirect to "/digihaler/request," which is a POST URL that displays a thank you template.

[the remark this answers is not in the document]
Comment (JwalantaPhil) [reply netlify.toml:96]: what's the url of the page that has the form?

[the remark this answers is not in the document]
Comment (hastagurung) [reply netlify.toml:96]: https://enroll.phil.us/digihaler

[the remark this answers is not in the document]
Comment (JwalantaPhil) [reply netlify.toml:96]: then this is ok

**Review by JwalantaPhil — APPROVED** [review]

**Pull request merged**